### PR TITLE
Different hotfixes

### PR DESCRIPTION
**Pull request by NikKovIos**
Fixed the filters bug in simulator when not resized the thumbnail properly.
Added a loader to the filters to prevent multiple touches.
Bottom pager title now autoresized.
Curtain - increased alpha Bottom bar - height reduced to 44px like instagramm Smoother expanding of the bottom gallery.
MediaManager - Added a forse cancelling of video exporting if picker is cancelled. Fixed glitching when processing the multiple videos.
Added the minimum width property to restrict the big height images like in insta!
Fixed Filters example.
Cover in video now showing properly.
Now can add videos without audio.
Renamed VideoVC to VideoCaptureVC. Same with Helper.

### CODE REVIEW

File: Source/BottomPager/YPMenuItem.swift [hunk 1]
```diff
@@ -37,12 +37,15 @@ final class YPMenuItem: UIView {
         )
         
         textLabel.centerInContainer()
```
Comment (s4cha) [inline Source/BottomPager/YPMenuItem.swift:39]:
```swift
textLabel.centerInContainer()
|-(10)-textLabel-(10)-|
```
to
```swift
textLabel.centerVertically()
|-(10)-textLabel-(10)-|
```

File: Source/Pages/Gallery/YPAssetZoomableView.swift [hunk 2]
```diff
@@ -59,63 +62,64 @@ final class YPAssetZoomableView: UIScrollView {
...
+
             // Fit video view if only squared
             if YPConfig.library.onlySquare {
                 strongSelf.fitImage(true)
             }
             
             // Stored crop position in multiple selection
-            if let scp = storedCropPosition {
-                strongSelf.applyStoredCropPosition(scp)
+            if let scp173 = storedCropPosition {
```
Comment (s4cha) [inline Source/Pages/Gallery/YPAssetZoomableView.swift:84]: 173?

File: Source/Pages/Gallery/YPAssetZoomableView.swift [hunk 3]
```diff
@@ -124,40 +128,58 @@ final class YPAssetZoomableView: UIScrollView {
...
```
Comment (s4cha) [inline Source/Pages/Gallery/YPAssetZoomableView.swift:131]: I'm curious as to why 173?

File: Source/BottomPager/YPBottomPagerView.swift [hunk 4]
```diff
@@ -27,7 +27,7 @@ final class YPBottomPagerView: UIView {
             0,
             |scrollView|,
             0,
-            |header| ~ 50
+            |header| ~ 44
```
Comment (s4cha) [inline Source/BottomPager/YPBottomPagerView.swift:30]: 👍

**Review by s4cha — APPROVED** [review]

File: Source/Pages/Gallery/YPAssetZoomableView.swift (hunk 2 above)
Comment (NikKovIos) [reply Source/Pages/Gallery/YPAssetZoomableView.swift:84]: It was an easter egg) http://www.scp-wiki.net/scp-173

File: Source/BottomPager/YPBottomPagerView.swift (hunk 4 above)
Comment (NikKovIos) [reply Source/BottomPager/YPBottomPagerView.swift:30]: It should be 50 - i've compared with Insta

Comment (s4cha) [reply Source/BottomPager/YPBottomPagerView.swift:30]: Yes perfect :)

File: Source/Pages/Gallery/YPAssetZoomableView.swift (hunk 2 above)
Comment (s4cha) [reply Source/Pages/Gallery/YPAssetZoomableView.swift:84]: lol didn't know about that xD

**Pull request merged**